0000-template.md

```diff
@@ -0,0 +1,56 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- CppMicroServices Issue: (leave this empty)
```

Comment:
I didn't see any mention in RFC process when the ```RFC PR``` and ```CppMicroServices Issue``` entries are meant to be populated.